modules/serverless-kn-func-run.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * serverless/cli_tools/kn-func-ref.adoc
+// * serverless/functions/serverless-functions-getting-started.adoc
+
+:_content-type: REFERENCE
+[id="serverless-kn-func-run_{context}"]
+= Running a function locally
+
+You can use the `kn func run` command to run a function locally in the current directory or in the directory specified by the `--path` flag:
+
+.Example command to run a function in the current directory
```

Comment:
I think these should be formatted like this so the command goes in under the step. It looks like it renders correctly though so I am not sure.

```suggestion
.Example command to run a function in the current directory
+
[source,terminal]

----
$ kn func run
----
```

Reply:
I think I'll leave it as is if that's OK, since this is the same way other similar modules are formatted and there are no rendering issues.